app/helpers/application_helper.rb

```diff
@@ -10,4 +10,8 @@ def resource
   def devise_mapping
     @devise_mapping ||= Devise.mappings[:user]
   end
+
+  def three_month(user)
```

Comment:
rework it. pass number of months to method. rename method.

and if you passing "user" => Company - it's a sample of bad naming for variable. Must be more abstract

Reply:
Done.